docs/core/additional-tools/dotnet-core-uninstall.md

```diff
@@ -0,0 +1,339 @@
+---
+title: .NET Core Uninstall Tool
+description: An overview of the .NET Core Uninstall Tool, a guided tool that enables the controlled clean-up of .NET Core SDKs and Runtimes.
+author: yuchong-pan
+ms.date: 08/06/2019
+---
+# .NET Core Uninstall Tool
+
+The [.NET Core Uninstall Tool (`dotnet-core-uninstall`)](https://dotnet.microsoft.com/download/dotnet-core/uninstall-tool) lets you clean-up .NET Core SDKs and Runtimes on a system such that only the desired versions remain. A collection of options is available to specify which versions you want to uninstall.
+
+The tool supports Windows and macOS. Linux is currently not supported.
+
+Note that the Windows version of this tool can only uninstall .NET Core SDKs and Runtimes that were installed using .NET Core SDK or Runtime installers. Prior to Visual Studio 2019 Update 16.3, the Visual Studio installer called the .NET Core SDK installer and these can be uninstalled with the tool. The `dotnet-core-uninstall list` command displays which SDKs can be uninstalled with the tool.  
+
+## How to use the tool
+
+1. Download and install the tool.
+2. Display installed .NET Core SDKs and Runtimes that can be removed by this tool.
+3. Do a dry run to ensure the right things will be uninstalled, and uninstall .NET Core SDKs and Runtimes by running the tool with Admin privileges.
+4. Optional: delete the `NuGetFallbackFolder` if you no longer need it.
+5. Remove the `dotnet-core-unintall`  tool.
+
+Read the following sections for details.
+
+### Installing
+
+You can download the .NET Core Uninstall Tool from [https://dotnet.microsoft.com/download/dotnet-core/uninstall-tool](https://dotnet.microsoft.com/download/dotnet-core/uninstall-tool) and put the single executable (`dotnet-core-uninstall.exe`) in a directory.
+
+__Note__: The tool requires elevation to uninstall .NET Core SDKs and Runtimes. Therefore, it should be installed in a write-protected directory such as `C:\Program Files` on Windows and `/usr/local/bin` on macOS. See also [Elevated access for dotnet commands](https://docs.microsoft.com/dotnet/core/tools/elevated-access).
```

Comment:
- Use the alert style note.
- Made directory italic
- `or` seems more suitable than `and`.
- Used relative path.

```suggestion
> [!NOTE]
> The tool requires elevation to uninstall .NET Core SDKs and Runtimes. Therefore, it should be installed in a write-protected directory such as *C:\Program Files* on Windows or */usr/local/bin* on macOS. See also [Elevated access for dotnet commands](../tools/elevated-access.md).
```